aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/GetSamplingRulesResponse.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package io.opentelemetry.contrib.awsxray;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+import java.util.List;
+import java.util.Map;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+@AutoValue
+abstract class GetSamplingRulesResponse {
```

Comment:
Not really an issue but I wonder if the aws sdk provides any of this and if adopting the dependency is undesirable overall.

Reply:
I wish the models were published separately but they are tied to the real SDK so would end up relatively heavyweight, and a particular problem is users using AWS SDK v1 prefer a library not to bring on v1 and vice versa. We got this report on the AWS x-ray SDK where we do use the AWS sdk for the models.

Stubs do get published which we may be able to autogen these classes with if we wanted to but not worth it for just these I guess.